docs/index.adoc

```diff
@@ -283,6 +343,17 @@ Example: **Difi/ID-porten/VER1**
 {zwsp} +
 
 == Upgrade Notes
+
+=== Version 3
+
+
+- User `displayName` and `email` are now automatically updated when the user logs in with Authorization Code Flow.
+- Authorization Code Flow with PKCE is now supported and used by default. Disable it (`usePkce = false`) if your provider does not support it.
+- In Authorization Code Flow, the ID Token versification is now done with the public keys from the JWKS URI (`jwksUri`). Since this field is optional the verification will not be done if the field is not set. Note, that if `oidcWellKnownEndpoint` is set, the `jwksUri` will be fetched from the well-known endpoint automatically.
```

Comment:
```suggestion
- In Authorization Code Flow, the ID Token verification is now done with the public keys from the JWKS URI (`jwksUri`). Since this field is optional the verification will not be done if the field is not set. Note, that if `oidcWellKnownEndpoint` is set, the `jwksUri` will be fetched from the well-known endpoint automatically.
```